### ISSUE TITLE
Simple Example fails

### ISSUE THREAD

**Issue by pjz** [AspenWeb/pando.py#580]
```
$ python -m pando
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 163, in _run_module_as_main
    mod_name, _Error)
  File "/usr/lib/python2.7/runpy.py", line 111, in _get_module_details
    __import__(mod_name)  # Do not catch exceptions initializing package
  File "/tmp/p/local/lib/python2.7/site-packages/pando/__init__.py", line 62, in <module>
    import aspen.utils as _x  # this registers the 'repr' codec error strategy
ImportError: No module named utils
```

**Comment by Changaco** [comment]
Hey @pjz. The problem is that the current release of pando (0.44) asks for `aspen>=1.0rc1` when in fact it needs `aspen==1.0rc1`.

**Comment by pjz** [comment]
oh, should I be using HEAD instead?

**Comment by Changaco** [comment]
You can fix the problem with a `pip install aspen==1.0rc1`.

**Comment by Changaco** [comment]
Pando 0.45, which I released a few hours ago, fixes this issue.
